docs-new/docs/graphql/core/api-reference/schema-metadata-api/custom-functions.mdx

```diff
@@ -121,28 +119,22 @@ the context of the resulting GraphQL API.
 One such use case might be a function that wraps a simple query and
 performs some logging visible only to administrators.
 
-<div class="note">
-
-<div class="title">
-
-Note
-
-</div>
+:::info Note
 
 It's easy to accidentally give an SQL function the wrong volatility (or
 for a function to end up with `VOLATILE` mistakenly, since it's the
 default).
 
-</div>
+:::
 
-### Args syntax
+### Args syntax {#schema-metadata-track-function-v2-syntax}
```

Comment:
id mismatch - sorry missed this in yeserday's review comments
```suggestion
### Args syntax {#schema-metadata-track-function-syntax-v2}
```

Reply:
Done